### DISCUSSION TITLE
Filter rules for a role don't show items

### DISCUSSION THREAD

**Discussion by fatoon29** [directus/directus#6367] [Archive]
I create a filter to show specific items to a role : 

```json
{
    "server": {
        "name": {
            "_eq": "S1"
        }
    }
}
```

When i connect with a user assigne to this role i can see  1-18 of 18items, thats the correct number of items with the filter, but no items is shown.

![role](https://user-images.githubusercontent.com/55839100/122541084-8cd5f200-d029-11eb-9c21-addb70e09384.png)

**Comment by hugopvl** [comment]
```
{
        {
            "server": {
                "_eq": "S1"
            }
        }
		 
}
```
or
```
{
        {
            "name": {
                "_eq": "S1"
            }
        }
		 
}
```

**Reply by hugopvl** [reply]
I'm sorry for my answer above, I just noticed that you had a relational filter.

thanks

**Comment by philleepflorence** [comment]
What is the `status` for the 18 items and permission for that role?

**Reply by fatoon29** [reply]
i give the permission to create and to see items that's on S1.
What you mean by status, if they are published?

**Reply by philleepflorence** [reply]
Yes, publish...
Hard to tell why the items don't show by the image and filter alone.
